### DISCUSSION TITLE
.netFramework features

### DISCUSSION THREAD

**Discussion by MfanaShezi** [codewars/codewars.com#2712] [Q&A]
does code wars have the  .netframe work features? and how can i add  using directive since i'm not given the main method?

**Comment by DeoEsor** [comment]
> does code wars have the .netframe work features?

I'm there (at codewars) about a month and I use some .NET Core features, so I suppose that .NET Framework features available too.

> and how can i add using directive since i'm not given the main method?

I'm just adding using directive in beginning of file like that
![image](https://user-images.githubusercontent.com/60220969/176269783-2bae8f13-89a6-4ede-85ee-40c0d8583f13.png)

UPD: I strongly recommended you adding with using directive all namespaces that you are using
